### PR TITLE
Import necessary BGFX libraries Trial 2

### PR DESCRIPTION
**Pull request by lilseyi**

### CODE REVIEW

File: shared/SampleShared/SampleShared_win32.vcxproj.filters [hunk 1]
```diff
@@ -39,6 +39,5 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
-    <None Include="$(OpenXRLoaderBinaryRoot)\bin\openxr_loader.dll" />
```
Comment (jamesdooleymsft) [inline shared/SampleShared/SampleShared_win32.vcxproj.filters:42]: Was this intended?

**Review by brycehutchings — APPROVED** [review]

File: shared/XrSceneLib/XrSceneLib_uwp.vcxproj.filters [hunk 2]
```diff
@@ -87,7 +87,5 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
-    <None Include="$(OpenXRLoaderBinaryRoot)\bin\openxr_loader.dll" />
```
Comment (jamesdooleymsft) [inline shared/XrSceneLib/XrSceneLib_uwp.vcxproj.filters:90]: Is this intended?

**Review by jamesdooleymsft — CHANGES_REQUESTED** [review]

File: shared/SampleShared/SampleShared_win32.vcxproj.filters (hunk 1 above)
Comment (brycehutchings) [reply shared/SampleShared/SampleShared_win32.vcxproj.filters:42]: This tends to flip-flop due to the NuGet package quirky behavior. It should be okay.

File: shared/XrSceneLib/XrSceneLib_uwp.vcxproj.filters (hunk 2 above)
Comment (brycehutchings) [reply shared/XrSceneLib/XrSceneLib_uwp.vcxproj.filters:90]: It's ok

**Review by jamesdooleymsft — APPROVED** [review]

**Pull request merged**